tai64/src/lib.rs

```diff
@@ -40,6 +43,11 @@ const NANOS_PER_SECOND: u32 = 1_000_000_000;
 #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
 pub struct TAI64(pub u64);
 
+#[cfg(feature = "zero_out")]
+impl Zeroize for TAI64 {
+    fn zeroize(&mut self) {}
```

Comment:
This method doesn't do anything as-is: `Zeroize` isn't just a marker trait.

You'll need to do:

```suggestion
    fn zeroize(&mut self) {
        self.0.zeroize();
    }
```